main/libgsasl/APKBUILD

```diff
@@ -17,6 +17,7 @@ _builddir="$srcdir"/$pkgname-$pkgver
 
 build() {
 	cd "$_builddir"
+	update_config_guess || return 1
```

Comment:
This does not belong to the build phase, it should be called from `prepare()` function:

```sh

prepare() {
	default_prepare || return 1
	update_config_guess
}
```